### PR TITLE
Stack pages Update

### PR DESCRIPTION
**Pull request by alem355**

### CODE REVIEW

File: content/en/stack/_index.md [hunk 1]
```diff
@@ -46,11 +46,7 @@ All categories except `research` can work in production.
 * Licensing policy
 
 ### Technology Committee
-The committee is a community of enterprise architects, technical architects and technical leaders of development teams
-and IT infrastructure. The main goal of the committee is the selection and recommendations on the use of technologies
-in Leroy Merlin Russia and increasing the efficiency of use of company resources.
-Technologies are understood as programming languages, frameworks and libraries, software tools and approaches
-in the context of specific usage scenarios.
+The committee is a community of enterprise architects, technical architects and technical leaders of development teams and IT infrastructure. The main goal of the committee is the selection and recommendations on the use of technologies in Leroy Merlin Russia and increasing the efficiency of use of company resources. Technologies are understood as programming languages, frameworks and libraries, software tools and approaches in the context of specific usage scenarios.
```
Comment (aleksandrregner) [inline content/en/stack/_index.md:49]: at Leroy Merlin Russia
The main goal is.. and then you have a list of two items

File: content/en/stack/comittee.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: "Технологический комитет"
+weight: 30
+type: docs
+---
+
+# What is technology committee
+
+The committee is a community of enterprise architects, technical architects, and technical leaders of development teams and IT infrastructure. The main goal of the committee is to select and recommend the use of technologies in the Leroy Merlin Russia and to improve the efficiency of using the company's resources. Technologies mean programming languages, frameworks and libraries, software tools, and approaches in the context of specific use cases.
```
Comment (aleksandrregner) [inline content/en/stack/comittee.md:9]: at Leroy Merlin Russia
The definition differs from the definition from stack/_index.md

File: content/en/stack/_index.md (hunk 1 above)
Comment (aleksandrregner) [reply content/en/stack/_index.md:49]: technical leaders of development and IT infrastructure teams

**Review by aleksandrregner — CHANGES_REQUESTED** [review]

File: content/en/stack/comittee.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: "Технологический комитет"
```
Comment (voborgus) [inline content/en/stack/comittee.md:2]: Change the title please

File: content/en/stack/how-to-start.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+title: "How to start to use a new technology?"
+weight: 20
+type: docs
+---
```
Comment (voborgus) [inline content/en/stack/how-to-start.md:5]: Please fill the document with info from the main page

File: content/ru/stack/comittee.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Technology Comittee"
```
Comment (voborgus) [inline content/ru/stack/comittee.md:2]: Please translate the title

File: content/ru/stack/how-to-start.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+---
+title: "Как начать использовать новую технологию?"
+weight: 20
+type: docs
+---
```
Comment (voborgus) [inline content/ru/stack/how-to-start.md:5]: Please fill the document from the main page

**Review by voborgus — CHANGES_REQUESTED** [review]

**Review by voborgus — APPROVED** [review]

**Pull request merged**